### DISCUSSION TITLE
How do I get a nice summary of warnings like what revive does?

### DISCUSSION THREAD

**Discussion by MichaelSnowden** [golangci/golangci-lint#3430] [Q&A] [answered]
In revive, I can use `-formatter friendly` to get a summary of warnings like this:

```
⚠ 3051 problems (0 errors, 3051 warnings)

Warnings:
  2441  exported             
  311   var-naming           
  181   package-comments     
  61    unexported-return    
  19    indent-error-flow    
  17    increment-decrement  
  8     var-declaration      
  8     receiver-naming      
  2     blank-imports        
  2     error-strings        
  1     context-as-argument  
```

I am currently using revive in golangci-lint, but I don't see any `out-format` option that does something like this. How do I get a summary of warnings and errors using golangci-lint?

**Comment by ldez** [comment, marked as answer]
Hello,

An option has been added in v1.56.0: `show-stats`

https://golangci-lint.run/usage/configuration/#output-configuration

**Comment by k3n** [comment]
Hi! Is it possible to get this output with a CLI flag?

**Reply by ccoVeille** [reply]
You mean this ?

https://github.com/golangci/golangci-lint/discussions/3430#discussioncomment-9135624

**Reply by k3n** [reply]
Thanks! I must be doing something wrong...

```sh
$ golangci-lint --show-stats
Error: unknown flag: --show-stats
```

I think I'm up-to-date?

```sh
$ golangci-lint --version
golangci-lint has version 1.59.0 built with go1.22.3 from 2059b18 on 2024-05-25T11:38:08Z
```

**Reply by k3n** [reply]
Ah yup I'm a dummy!!

```sh
$ golangci-lint run --show-stats
```
